recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
 
 build:
   number: 0
-  skip: True  # [py3k]
+  skip: True  # [osx and py3k]
```

Comment:
> Are you familiar with conda-smithy to do that?

I am not... Google brought up [this](https://github.com/conda-forge/staged-recipes/wiki/conda-smithy-rerender), so should I do this locally and add the result to the PR?